src/applications/combined-debt-portal/debt-letters/components/HowDoIPay.jsx

```diff
@@ -86,21 +86,34 @@ const HowDoIPay = ({ userData }) => (
           <va-telephone contact="6127136415" international />.
         </p>
       </va-accordion-item>
-      <va-accordion-item header="Option 3: Pay by mail" id="third">
-        <p className="vads-u-margin-y--0">
-          Find instructions on how to pay by mail in the demand letter sent to
-          your address or you can
-          <Link className="vads-u-margin-left--0p5" to="/debt-balances/letters">
-            download them online
-          </Link>
-          .
-        </p>
-      </va-accordion-item>
+      {showDebtLetterDownload ? (
+        <va-accordion-item header="Option 3: Pay by mail" id="third">
+          <p className="vads-u-margin-y--0">
+            Find instructions on how to pay by mail in the demand letter sent to
+            your address or you can
+            <Link
+              className="vads-u-margin-left--0p5"
+              to="/debt-balances/letters"
+            >
+              download them online
+            </Link>
+            .
```

Comment:
**Nitpick:** It's slightly better for accessibility to include a trailing period like this in the link. Otherwise, it's read out separately on screen readers.

Reply:
Oh I didn't know that! Updating now and I'll keep it in mind moving forward!